Fitbit.Portable.Tests/SleepTests.cs

```diff
@@ -14,6 +14,8 @@ namespace Fitbit.Portable.Tests
     [TestFixture]
     public class SleepTests
     {
+        #region Success Test
```

Comment:
Understandable, i can remove the regions this evening, better off having things consistent also